useful.rb

```diff
@@ -0,0 +1,42 @@
+def jruby?; RUBY_ENGINE == "jruby"; end
+
+def mri?; RUBY_ENGINE == "ruby"; end
+
+def legacy?
+  Gem::Version.new(RUBY_VERSION) > Gem::Version.new("2.0.0")
+end
+
+def modern?; !legacy?; end
+
+def two_one?
+  Gem::Version.new(RUBY_VERSION) <= Gem::Version.new("2.1.0")
+end
```

Comment:
It looks like Rubocop doesn't like one-line method definitions (`jruby?`, `mri?`, `modern?`). Breaking these up over three lines should make the linter happy -- you can run `rubocop --auto-correct` to make these changes automatically.